packages/core/helper-plugin/lib/src/hooks/useAPIErrorHandler/utils/formatAPIError/index.js

```diff
@@ -0,0 +1,18 @@
+import { normalizeAPIError } from '../normalizeAPIError';
+
+export function formatAPIError(error, { formatMessage, intlMessagePrefixCallback }) {
```

Comment:
if you're not using this anywhere else, why not just include it in the hook?

Reply:
Good catch. Will make it private - at first I thought I might need it somewhere.